src/game.cpp

```diff
@@ -1788,7 +1789,12 @@ void Game::run()
 		updateProfilers(runData, stats, draw_times, dtime);
 		processUserInput(&flags, &runData, dtime);
 		// Update camera before player movement to avoid camera lag of one frame
-		updateCameraDirection(&cam_view, &flags);
+		updateCameraDirection(&cam_view_target, &flags);
+		float cam_smoothing = 1 - g_settings->getFloat("camera_smoothing");
```

Comment:
Either this needs to be `MYMAX(0.1f, 1 - g_settings->getFloat("camera_smoothing"); // or something similar`, or the acceptable range for cam(era)_smoothing limited in some other way.